app/controllers/profile_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ProfileController < ApplicationController
+  respond_to :html
+
+  def edit
+  end
+
+  def update
+    if current_user.update(user_params)
+      redirect_to edit_profile_path,
+                  notice: 'Профиль пользователя успешно обновлен.'
+    else
+      respond_with @user
+    end
+  end
+
+  def destroy
+    current_user.destroy
+    redirect_to login_path, notice: 'Пользователь успешно удален.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.